test/plugins/features/movies/controller.test.js

```diff
@@ -16,4 +16,40 @@ describe('movie controller', () => {
 
   });
 
+  describe('get', () => {
+
+    it('returns a list of movies if no query params are passed', async () => {
+      const movies = await Controller.get();
+      expect(movies.models['0'].attributes.title).to.eql('180');
+      expect(movies.models['1'].attributes.title).to.eql('24 Hours on Craigslist');
+    });
+
+    it('returns a list of one movie if it is an exact title match to a query parameter', async () => {
+      const movies = await Controller.get({ title: '180' });
+      expect(movies.models['0'].attributes.title).to.eql('180');
+      expect(movies.models['1']).to.be.an('undefined');
```

Comment:
Is this to check how many movies we got back? If so, I think you can check the length of `movies.models` instead.